dask/dataframe/io/parquet/core.py

```diff
@@ -95,6 +95,64 @@ def __call__(self, part):
         )
 
 
+ParquetFunctionWrapper = ReadParquetFunctionWrapper
```

Comment:
Why is the alias necessary?

Reply:
We could just use `ParquetFunctionWrapper` and `ToParquetfunctionWrapper` and avoid this alias.  Does that sound reasonable?

Background: I just liked the "Read*" vs "To*" distinction in the names, and the only reason I used an alias after changing the "Read*" name is that `ParquetFunctionWrapper` is techincally a public API.

Reply:
Update: I removed the alias, so we just have `ParquetFunctionWrapper` and `ToParquetfunctionWrapper` for now.